notification-stream/src/providers/pushly.ts

```diff
@@ -0,0 +1,4 @@
+export const createPushlyNotification = ((message: string, vfUserId ? : string) => {
+
+
+});
```

Comment:
Also can you update your linter settings to have an empty line at the end of js/ts files?